Tests/update_content_data.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import os
+import ast
+import demisto_client
+
+from Tests.test_utils import print_error
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Utility to upload new content')
+    parser.add_argument('-u', '--user', help='The username for the login', required=True)
+    parser.add_argument('-p', '--password', help='The password for the login', required=True)
+    parser.add_argument('-s', '--server', help='The server to connect to (leaving out the protocol e.g.'
+                        ' without \'https://\')', required=True)
+    parser.add_argument('-up', '--content_zip', help='Path to new content zipfile to upload', required=True)
+
+    options = parser.parse_args()
+
+    return options
+
+
+def upload_content(server, username, password, content_zip_path):
```

Comment:
add docstring, google style.
please specify that it updates content rather then uploads it, maybe also worth changing the function name here